app/languages/definitions/Malayalam.yml

```
@@ -0,0 +1,14 @@
+locale: ml
+dictionaryFile: ml-utf8.csv
+abcString: xyz
```

This is not truly Malayalam but rather a mixture of English and Malayalam. It should have a different locale and name, something like Hinglish, which is a mixture of Hindi and English.

Also, the locale must consist of four letters, not two. English is the only exception.

Let's do the following:
```suggestion
locale: ml-GB
name: Malayalam / Latin
dictionaryFile: malayalam-latin-utf8.csv
```

Let's as well rename the `.yml` file to `MalayalamLatin.yml` for consistency with Tamazight Latin.

Also, must add icons.